ephios/core/templates/core/disposition/fragment_participation.html

```diff
@@ -57,13 +60,23 @@ <h6 class="d-inline">
             <div class="card-body">
                 {{ form.individual_start_time|as_crispy_field }}
                 {{ form.individual_end_time|as_crispy_field }}
-                {% if form.comment.initial %}
-                    {{ form.comment|as_crispy_field }}
+                {% if form.previous_comments %}
+                    {{ form.previous_comments|as_crispy_field }}
                 {% endif %}
+                <div class="mb-3">
+                    <div class="form-label">
+                        {{ form.comment.label_tag }}
+                        <span class="float-end form-check">
+                            <input type="checkbox" id="{{ form.comment_is_internal.auto_id }}" name="{{ form.comment_is_internal.html_name }}" class="form-check-input">
+                            {{ form.comment_is_internal.label_tag }}
```

Comment:
This does show a colon at the end, which I think doesn't fit here:
![image](https://github.com/user-attachments/assets/6190bb7b-4bc2-4829-96f5-5bd1ef13e53c)